src/airport.js

```diff
@@ -1 +1,91 @@
+class Airport {
+
+    constructor() {
+
+        this.planesInAirport = [];
+        this.airportCapacity = 10; // default 
+
+    }
+
+
+    landPlane(plane) {
+
+        if (this.isAirportFull()) {        //checks airport is not full
+            return console.log('Airport is full')
+        }
+
+        if (this.checkAirport(plane)) { return console.log('Plane has already landed') }          //checks plane doesn't already exist
+
+
+        if (this.validID(plane)) {  // checks plane is a valid string
+            this.planesInAirport = [...this.planesInAirport, plane]
+        }
+        else {
+            return console.log('Invalid ID ')
+        }
+
+
+    }
+
+
+    checkAirport(plane) {
+
+        if (this.planesInAirport.includes(plane)) {
+            return true
+        }
+        else { return false }
+    }
+
+
```

Comment:
In the 'checkAirport' and 'validID' method, I could have tried to use a ternary operator instead of an if else statement.